instructions-for-devs.md

```diff
@@ -142,12 +144,15 @@ host = "localhost"
 
 [p2p]
 listen = true
-port = 8885
-#make sure this port is reachable from the internet
+port = 8885 # Make sure this port is reachable from the internet
+# A string array of reachable socket addresses for this node, port defaults to 8885 if unspecified
```

Comment:
Shouldn't the default port be the config value `p2p.port`?

Reply:
That would be good, yes.